src/libraries/System.Net.Http/src/System/Net/Http/FormUrlEncodedContent.cs

```diff
@@ -27,7 +27,7 @@ private static byte[] GetContentByteArray(IEnumerable<KeyValuePair<string?, stri
 
             // Encode and concatenate data
             StringBuilder builder = new StringBuilder();
-            foreach (KeyValuePair<string?, string?> pair in nameValueCollection)
+            foreach (KeyValuePair<string, string?> pair in nameValueCollection)
```

Comment:
Should we throw an exception when key is null?

Reply:
Let's leave it as it is for now - it is the same behavior as in 3.1.